### DISCUSSION TITLE
How to add buttons that link to files with button color that match the theme?

### DISCUSSION THREAD

**Discussion by junyuan-chen** [alshedivat/al-folio#1715] [Q&A]
This is a novice question. I want to add something like the outline buttons shown here: https://getbootstrap.com/docs/5.2/components/buttons/#outline-buttons

However, if I directly put
```
<a href="https://www.google.com/">
  <button type="button" class="btn btn-outline-primary">Button</button>
</a>
```
into a markdown file. Then, the color of the button will have a different color than the theme color. But, then the button has a sharp corners that look different from what I see from the bootstrap website. The text on the button also get capitalized.

How do I change the color of the button, get the corner of the button rounded, and also avoid capitalizing the text on button?

Thank you!

**Comment by george-gca** [comment]
Probably the classes' properties are being overwritten somewhere. Check in one of the files in `_sass/`. The easiest way to debug this is by using the developer tools available in [Chrome](https://developer.chrome.com/docs/devtools/overview/) or [Firefox](https://firefox-source-docs.mozilla.org/devtools-user/).
